server.js

```diff
@@ -150,6 +151,12 @@ async function work(body) {
     return;
   }
 
+  if(skipTitle == "WIP"){
```

Comment:
nit: put a space after `if`, use `===`, single quote `'WIP'` and space before `{`:

```
if (skipTitle === 'WIP') {
```